articles/tech-resources/custom-bounce-domain.md

```diff
@@ -3,16 +3,22 @@ title: "Creating a Custom Bounce Domain"
 description: "You can improve your deliverability and email branding by adding a custom bounce domain A custom bounce domain provides another layer to the branding that some IS Ps care about Note Only one custom bounce domain is permitted per account at this time By adding a bounce domain to your..."
 ---
 
-You can improve your deliverability and email branding by configuring each of your sending domains as bounce domains. A bounce domain provides another layer to the branding that some ISPs care about.
+By creating a custom bounce domain, you can customize the address that is used for the Return-Path header (which denotes the destination for out-of-band bounces). This bounce domain overrides the default Return-Path (also known as the envelope FROM) value of "sparkpostmail.com" for all messages sent.
 
-By using your sending domain as your bounce domain, you can customize the address that is used for the Return-Path header (which denotes the destination for out-of-band bounces). This bounce domain overrides the default Return-Path (also known as the envelope FROM) value of "sparkpostmail.com" for all messages sent.
+Please note the following:
 
-To configure a bounce domain, choose the subdomain you would like to use (e.g. example.com) and add the following CNAME record to your DNS settings:
+1. Only subdomains can be configured as bounce domains (e.g. “bounce.example.com”)
+
+1. Root domains can only be configured as sending domains (e.g. “example.com"). Adding a CNAME record for your root domain will cause your website to stop working, so please ignore this CNAME verification step in your sending domain configuration.
```

Comment:
So this isn't a true statement. CNAME records won't cause their website to stop working. What I would say instead: we do NOT recommend using a root domain for sending email. Instead create something like email.example.com. You can use this as both the sending and bounce domain. In order to set it up as a bounce domain, you will need to add the CNAME record in your DNS. If you prefer to use your root domain as the sending domain, you can do that and simply not add the CNAME record to the root example.com domain.